.eslintrc

```diff
@@ -1,3 +1,13 @@
 {
-    "extends": "ebay"
-}
\ No newline at end of file
+    "extends": "ebay",
+    "rules": {
+         "import/order": [2, {"groups": ["builtin", "external", "internal", "parent", "sibling", "index"]}],
+        "no-param-reassign": 0
```

Comment:
I think that `no-param-reassign` (currently set as `2`) in keeping a parameter immutable is helpful in preventing bugs. The current rule does allow object property to be reassigned.

https://github.com/eBay/eslint-config-ebay/blob/master/rules/bp.js#L90

https://github.com/eBay/eslint-config-ebay/commit/bb4b09d7b869251cd958b844ade6130eff2d9d70

Reply:
I agree with this. We should keep it as such.  Also makes it easier to have a more functional approach